### ISSUE TITLE
Do not show focus on Form elements that are not inputs that can be typed in

### ISSUE THREAD

**Issue by janechu** [microsoft/fast-dna#1950]
All elements that are not textarea or inputs with type text or number, should not show the focus styling on click.

See how focus visible is used on fast-components-styles-msft components. There should be a utility being used there on components such as Button and Checkbox.
